addOns/fuzz/src/main/resources/org/zaproxy/zap/extension/fuzz/resources/Messages.properties

```diff
@@ -484,3 +480,7 @@ fuzz.httpfuzzer.processor.tagcreator.validation.messageboxtitle = Please verify
 fuzz.httpfuzzer.processor.tagcreator.validation.textfieldsarerequired = All text fields in the selected group are required.
 fuzz.httpfuzzer.processor.tagcreator.validation.regexsyntaxerror = The entered regular expression has a syntax error:\r\n{0}
 fuzz.httpfuzzer.processor.tagcreator.validation.regexatleastonegroup = The entered regular expression needs at least one group.
+
+fuzz.api.action.multiplePayloadFuzzer = Upload the JSON file containing fuzz, payload and processors locations, the schema can be obtained from zap-api-docs.
+fuzz.api.action.simpleHttpFuzzer = A Simple Http Fuzzer fuzzes one location, with inputs messageId {int}, requestLocation {body or header}, fuzzLocation, character at {int start: int end}, payload {Path of the file containing payloads}. Example: messageID: 7, requestLocation: "body", fuzzLocation: "4:8", payload: "/home/users/payloads.txt"
```

Comment:
`A simple Http Fuzzer which fuzzes one location...` add "which"..."Fuzzer fuzzes" just looks funny, not grammatically wrong, just looks out of place.